sway-core/src/language/ty/declaration/declaration.rs

```diff
@@ -780,7 +780,7 @@ impl TyDecl {
         }
     }
 
-    /// Friendly type name string used for various reporting,
+    /// Friendly type name string used for various reportings,
```

Comment:
```suggestion
    /// Friendly type name string used for various reporting,
```

`reporting` is a verb, not a noun.

Reply:
"To report/reporting" is a verb for sure, but it is a noun as well. E.g., Cambridge Dictionary [lists "reporting" first as a noun](https://dictionary.cambridge.org/dictionary/english/reporting). Also, a search for the exact phrase ["various reportings"](https://www.google.com/search?q=%22various+reportings%22) gives quite some results. That's why I would leave it as is.